scripts/test-coverage-xml.sh

```diff
@@ -7,8 +7,12 @@ set -x
 # test-coverage-xml.sh unit
 # For only integration tests call
 # test-coverage-xml.sh integration
+# To store durations, add a fourth argument 'store-durations'
```

Comment:
Is this comment needed and accurate?

Reply:
The fourth arg seems like it's the 5th arg, but it does force the storage of the test durations. My comment below questions whether it's being used correctly.